.github/workflows/publish_images.yaml

```diff
@@ -50,3 +50,50 @@ jobs:
       - name: Display log on error
         if: failure()
         run: docker logs pulp
+
+  ssl_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download s6-overlay
+        run: |
+          wget https://github.com/just-containers/s6-overlay/releases/download/v1.22.1.0/s6-overlay-amd64.tar.gz
+      - name: Set version
+        run: |
+          version="${GITHUB_REF#refs/heads/}-https"
+          echo "Building and publishing $version."
+          echo "VERSION=$version" >> $GITHUB_ENV
+      - name: Build the images
+        run: |
+          docker build --build-arg SCHEME="https" --file pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:latest .
+          if [ "$VERSION" = "latest-https" ]; then
+            docker build --build-arg SCHEME="https" --file pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:https .
+            docker build --file pulp_galaxy_ng/Containerfile --tag pulp/pulp-galaxy-ng:https .
+            docker build --file pulp_galaxy_ng/Containerfile --tag pulp/pulp:https .
+          else
+            docker build --file pulp/Containerfile --tag pulp/pulp:$VERSION .
+          fi
```

Comment:
This looks wrong somehow. Is it right, that we need the ssl image to be called "latest" for this build?
But why build the same thing again in line 70 instead of just "docker tag" it to the new name.

Reply:
yes, it is hacky because the images point to latest
```
FROM pulp/pulp-ci-centos:latest
```
and I agree, I change the PR to use docker tag